cypress/integration/prescription-spec.js

```diff
@@ -68,3 +68,95 @@ describe('Prescription', () => {
     cy.url().should('include','form')
   })
 });
+
+describe('Affirmation Prescription', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://www.affirmations.dev', {
+      body: {
+        affirmation: "you're amazing"
+        }
+      }
+    })
+    cy.visit('http://localhost:3000');
+    cy.get('.yes-button').click()
+    cy.get('form').find('.name-input').type('cypress')
+    cy.get('form').find('.forward-button').as('next').click()
+    cy.get('form').find('.name-input').type('cypress')
+    cy.get('@next').click()
+    cy.get('form').find('.problem-type-select').select("big bad bug")
+    cy.get('@next').click()
+    cy.get('form').find('.tech-type-select').select('everything')
+    cy.get('@next').click()
+    cy.get('@next').click()
+    cy.get('@next').click()
+    cy.get('form').find('.days-input').type('10')
+    cy.get('@next').click()
+    cy.get('form').find('.hours-input').type('1')
+    cy.get('@next').click()
+    cy.get('form').find('.progress-time-input').select('never')
+    cy.get('@next').click()
+    cy.get('form').find('.current-time-input').select('11')
+    cy.get('@next').click()
+    cy.get('form').find('.break-input').select('yesterday')
+    cy.get('@next').click()
+    cy.get('form').find('.eat-input').select('minutes')
+    cy.get('@next').click()
+    cy.get('form').find('.multi').select('adventurous')
+    cy.get('@next').click()
+    cy.get('form').find('.feeling-input').select('1')
+    cy.get('@next').click()
+    cy.get('form').find('.penult-button').click()
+    cy.get('form').find('.submit-button').click().wait(2000)
+  });
+
+  it('Should display the prescription', () => {
+    cy.get('.prescription-card'').find('.title').contains('a little love')
+    cy.get('.prescription-card'').find('.content').contains("you're amazing")
+  })
+});
+
+describe('Advice Prescription', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'some general advice', {
+      slip: {
+        advice: "just do it"
+        }
+      }
+    })
+    cy.visit('http://localhost:3000');
+    cy.get('.yes-button').click()
+    cy.get('form').find('.name-input').type('cypress')
+    cy.get('form').find('.forward-button').as('next').click()
+    cy.get('form').find('.name-input').type('cypress')
+    cy.get('@next').click()
+    cy.get('form').find('.problem-type-select').select("big bad bug")
+    cy.get('@next').click()
+    cy.get('form').find('.tech-type-select').select('everything')
+    cy.get('@next').click()
```

Comment:
repetitive because several input items must be changed to change the endpoint, and the form cannot be submitted unless all inputs are complete